src/Domain/Timetable/TimetableDayDateGateway.php

```diff
@@ -33,4 +33,12 @@ class TimetableDayDateGateway extends QueryableGateway
 
     private static $tableName = 'gibbonTTDayDate';
     private static $primaryKey = 'gibbonTTDayDateID';
+
+    public function deleteDates($firstDayOld, $firstDayNew)
```

Comment:
Looking good. Could we use a slightly more descriptive name here? Perhaps DeleteTimetableDatesInRange, or DeleteTTDayDatesBetweenDates. Thanks!